src/main/java/com/hyperwallet/clientsdk/model/HyperwalletVenmoAccount.java

```diff
@@ -15,9 +15,11 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public class HyperwalletVenmoAccount extends HyperwalletBaseMonitor {
 
+    public enum Status {ACTIVATED, INVALID, VERIFIED,DE_ACTIVATED}
+    public enum Type {VENMO_ACCOUNT}
```

Comment:
why do we need this to be public, I mean does this class represents another type in the future? `HyperwalletVenmoAccount` is always `VENMO_ACCOUNT` account right? please double check other FI objects as well

Reply:
moreover `HyperwalletVenmoAccount` type is always `VENMO_ACCOUNT` would it be safe and sound right that everytime we instantiate this construct the instance will always be of type `VENMO_ACCOUNT` ?

Reply:
Hi Peter, There are other resources like Bank Account which has multiple type like :
public static enum Type {BANK_ACCOUNT, WIRE_ACCOUNT}, and also previously these resources were taking the type from Transfer method which had all the Type clubbed together, so we created separate type in each resources and Setter method is required for the create resource API call...
public class HyperwalletTransferMethod extends HyperwalletBaseMonitor {

    public enum Type {BANK_ACCOUNT, WIRE_ACCOUNT, PREPAID_CARD, BANK_CARD, PAPER_CHECK, PAYPAL_ACCOUNT, VENMO_ACCOUNT}